recaf-ui/src/main/java/me/coley/recaf/config/ConfigRegistry.java

```diff
@@ -47,6 +49,10 @@ public class ConfigRegistry {
 		registerTypeAdapter(builder, WritableLongValue.class, (w, v) -> w.value(v.get()), r -> new SimpleLongProperty(r.nextLong()));
 		registerTypeAdapter(builder, WritableBooleanValue.class, (w, v) -> w.value(v.get()), r -> new SimpleBooleanProperty(r.nextBoolean()));
 		registerTypeAdapter(builder, WritableStringValue.class, (w, v) -> w.value(v.get()), r -> new SimpleStringProperty(r.nextString()));
+		// TODO: make enum adapters dynamically
```

Comment:
Are they already not handled? For instance `WorkspaceAction` usage in `DisplayConfig`

Reply:
These are enums in a ObjectProperty, so Gson cannot handle them, as it does not know the type of enum it's expecting. (I should probably use the new annotation for that?)

Reply:
Oh gotcha. Good point. Yeah go for it.